packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -456,6 +456,40 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         return numerator.div(FixedPoint.Unsigned(totalSupply())).rawValue;
     }
 
+    /**
+     * @notice Computes the current liquidity utilization ratio.
+     * @dev Used in computing realizedLpFeePct off-chain.
+     */
+    function liquidityUtilizationCurrent() public returns (uint256) {
```

Comment:
these methods were added to help off-chain computation with contract state data.